pkg/statsd/statsd_defaults_and_params.go

```diff
@@ -84,6 +84,8 @@ const (
 	DefaultBadLinesPerMinute = 0
 	// DefaultServerMode is the default mode to run as, standalone|forwarder
 	DefaultServerMode = "standalone"
+	// DefaultLogRawMetric is the default value for log-raw-metric flag
```

Comment:
```suggestion
	// DefaultLogRawMetric is the default value for whether to log the metrics received from network
```

Instead of referring to the _log-raw-metric flag_, I think it would improve readability if we included something explaining what this flag is doing